data/gml_registry.xsd

```diff
@@ -0,0 +1,69 @@
+                <xs:annotation>
+                  <xs:documentation>Definition of a feature type, with XML attributes to match given GML data and to define a .gfs or XML schema that contains the definition of the feature type. NOTE: The schema may contain definitions for multiple feature types.</xs:documentation>
+                </xs:annotation>
+                <xs:complexType>
+                  <xs:attribute name="elementName" type="xs:string" use="required">
+                    <xs:annotation>
+                      <xs:documentation>Name of the XML element that is used to identify the feature type. Typically the XML element name of an XML encoded feature. However, it can also be a different element that can be used in combination with @elementValue to produce a match for this feature type element.</xs:documentation>
+                    </xs:annotation>
+                  </xs:attribute>
+                  <xs:attribute name="elementValue" type="xs:string">
+                    <xs:annotation>
+                      <xs:documentation/>
+                      <xs:documentation>Used in combination with @elementName. If present, a match for this feature type element is only produced if an XML element whose name equals the value of @elementName and whose value equals the value of @elementValue is found in the GML data. For examle: If namespace/@prefix = 'ex', featureType/@elementName = 'typeOfFeature', and featureType/@elementValue = 'FT', then a match is produced if &lt;ex:typeOfFeature>FT&lt;/ex:typeOfFeature> is present in the GML data.</xs:documentation>
+                    </xs:annotation>
+                  </xs:attribute>
+                  <xs:attribute name="gfsSchemaLocation" type="xs:string">
+                    <xs:annotation>
+                      <xs:documentation>OGR .gfs file in which the feature type is defined (possibly together with other feature types). The GML driver will parse the layer definitions from that file. The schema location can be given as URL (http and https), absolute file path, and relative file path (relative to the location of the registry file).</xs:documentation>
+                    </xs:annotation>
+                  </xs:attribute>
+                  <xs:attribute name="schemaLocation" type="xs:string">
+                    <xs:annotation>
+                      <xs:documentation>XML schema file in which the feature type is defined (possibly together with other feature types). The GML driver will attempt to parse it and create layer definitions. The schema location can be given as URL (http and https), absolute file path, and relative file path (relative to the location of the registry file).</xs:documentation>
+                    </xs:annotation>
+                  </xs:attribute>
+                </xs:complexType>
+              </xs:element>
+            </xs:sequence>
+            <xs:attribute name="prefix" type="xs:string" use="required">
```

Comment:
It is actually optional, otherwise the data/gml_registry.xml file doesn't validate
```suggestion
            <xs:attribute name="prefix" type="xs:string" use="optional">
```

Would you mind also editing .github/workflows/validate_xml.yml to add validation of data/gml_registry.xml